### ISSUE TITLE
The UIA Name property is missing on the list element in selected PropertyGrid item

### ISSUE THREAD

**Issue by Vino-Wang** [dotnet/winforms#2802]
* .NET Core Version: 3.1.200-preview-014883

* Have you experienced this same bug with .NET Framework?: Yes

**Problem description:**
The Name property of the focusable element(list) is null.
![Actual](https://user-images.githubusercontent.com/18568603/73729765-9e8a6280-4770-11ea-8375-18b99c20d5a5.jpg)

**Expected behavior:**
The list element should have a meaningful name instead of null.

**Minimal repro:**
1. Create a WinForm 4.8 app with a PropertyGrid whose property items contain a list of strings or run the attached core app. [UIA_Name_missing.zip](https://github.com/dotnet/winforms/files/4152306/UIA_Name_missing.zip)
2. Run the Accessibility Insights for Windows tool to examine the UIA representation of the UI.
3. When keyboard focus is over in the right side of a property item, check the UIA tree.

**Comment by Tanya-Solyanik** [comment]
@vladimir-krestov  - naming suggestion and impact assessment from GuyBark:

> Bug 1060425 relates to a list in a WinForms PropertyGrid not having any UIA Name. By default, I would say that a nameless list does impact a product’s C grading, and it’s clearly information that’s required to meet MAS, (ie the name of important UI). That said, in this particular context, I think the severity of the bug is low. This bug relates to the list associated with an item in a PropertyGrid. So in this case, the customer navigates to the PropertyGrid, and then to some item within the PropertyGrid. Within that item, as I understand things, there’s one and only one list. It would be helpful for the list to have a name, so that the customer knows what the list relates to. I assume the name would be something like **“Values” or “Available Values”,** as it’s the list of values that the item can have. But given that it’s the only list available to the customer for the specific item, and the customer knows what item they’re at, once the customer’s familiar with the PropertyGrid experience, they’ll know that the list associated with an item is the list of available values for the item.
> 
> So I would say 1060425 is a MAS violation, but low severity given the specific context of the PropertyGrid. I do think it would be helpful for this to be fixed in WinForms, but I would not expect it to impact a products grade.

**Comment by Thea-Zhao** [comment]
Verified the issue with  Core 5.0.100-preview.3.20211.8 + VISX inserted in the Dev16 16.6.0, this issue is fixed.
![issue2802-Fixed](https://user-images.githubusercontent.com/53589182/79210517-d5789680-7df9-11ea-9b55-7a7e1849969e.png)

**Comment by Thea-Zhao** [comment]
@RussKie @vladimir-krestov Verified the issue with Core 5.0.100-preview.5.20221.6 + VISX inserted in the Dev16 16.7.0, this issue still reproduced. All the list elements have the same name property "Available values". Please see the following screenshot:
![repro1](https://user-images.githubusercontent.com/53589182/79963621-7b518400-843e-11ea-81ea-2990cb44707b.png)
![repro2](https://user-images.githubusercontent.com/53589182/79963629-7ee50b00-843e-11ea-9f01-c357ee858d8f.png)

**Comment by Thea-Zhao** [comment]
The validation result above is incorrect because we do not know that the "Available values" list are expected.
Verified with.NET Core 5.0.100-preview.5.20269.19 build, this issue is fixed. All the list elements have the same name property "Available values". This is expected that a PropertyGrid ComboBox entry should have "Available values" list in the accessibility tree.
![issue2802-fixed](https://user-images.githubusercontent.com/53589182/82429326-4a588380-9abe-11ea-9859-756f5385c095.png)
